tests/images/debian-systemd/files/bootstrap.sh

```diff
@@ -544,7 +544,7 @@ install_via_local_files() {
     find_then_install_deb "$INSTALL_SOURCEDIR" "tedge[_-]agent_*_$ARCH.deb"
     find_then_install_deb "$INSTALL_SOURCEDIR" "tedge[_-]apt[_-]plugin_*_$ARCH.deb"
     find_then_install_deb "$INSTALL_SOURCEDIR" "c8y[_-]configuration[_-]plugin_*_$ARCH.deb"
-    find_then_install_deb "$INSTALL_SOURCEDIR" "c8y[_-]log[_-]plugin_*_$ARCH.deb"
+    find_then_install_deb "$INSTALL_SOURCEDIR" "tedge[_-]log[_-]plugin_*_$ARCH.deb"
```

Comment:
We can simplify here as we never have **tedge_log_plugin**.
```suggestion
    find_then_install_deb "$INSTALL_SOURCEDIR" "tedge-log-plugin_*_$ARCH.deb"
```

Reply:
We'll leave this as is for the moment to concentrate on other work and this will eventually go away after combining the plugins.